base/operators.jl

```diff
@@ -1081,6 +1081,9 @@ is always a `Bool` and never `missing`.
 To determine whether an item is not in a given collection, see [`:∉`](@ref).
 You may also negate the `in` by doing `!(a in b)` which is logically similar to "not in".
 
+When broadcasting with `in` or `∈`, the collection will be broadcasted over unless the
+collection is wrapped in a tuple or a `Ref`.
```

Comment:
Maybe something like this? Or is it too verbose?
```suggestion
When broadcasting with `in.(items, collection)` or `items .∈ collection`, both
`item` and `collection` are broadcasted over, which is often not what is intended.
For example, if both arguments are vectors, the result is a vector indicating whether
each value in collection `items` is `in` the value at the corresponding position
in `collection`.
To get a vector indicating whether each value in `items` is in `collection`,
wrap `collection` in a tuple or a `Ref` like this:
`in.(items, Ref(collection))` or `items .∈ Ref(collection)`.
```
(Same below.)

Reply:
More verbose and explanatory is perfectly fine with me, but I've never contributed Base documentation before, so I'm not sure what the standard is. To make it even more clear should we insert a qualification about matching dimensions, like the following?

> if both arguments are vectors (and the dimensions match)